src/autoscaler/api/cred_helper/interface.go

```diff
@@ -1,9 +1,26 @@
 package cred_helper
 
-import "autoscaler/models"
+import (
+	"autoscaler/db"
+	"autoscaler/helpers"
+	"autoscaler/models"
+)
 
 type Credentials interface {
 	Create(appId string, userProvidedCredential *models.Credential) (*models.Credential, error)
 	Delete(appId string) error
 	Get(appId string) (*models.Credential, error)
+
+	// FIXME refactor
```

Comment:
So that we don't forget, we need to fix that

Reply:
@asalan316 do you fancy taking that one on?